src/core/geometry/qgsrectangle.h

```diff
@@ -43,7 +43,12 @@ class CORE_EXPORT QgsRectangle
   public:
 
     //! Constructor for a null rectangle
-    QgsRectangle() = default; // optimised constructor for null rectangle - no need to call normalize here
+    QgsRectangle()
```

Comment:
small nit: would not make it more sense to change the default initialization of those members around line 616 to those +/- <double>::max() values instead of their current 0 default ?

Reply:
Done with f61adf907ba146ee70099663e087dc422b29b22e